.idea/fileTemplates/MolgenisException.java

```diff
@@ -0,0 +1,44 @@
+#set( $EXCEPTION_CLASS_NAME = $Exception_name + "Exception")
+
+package $PACKAGE_NAME;
+
+import org.molgenis.i18n.CodedRuntimeException;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+/**
+* TODO
+*/
+// S2166 'Classes named like "Exception" should extend "Exception" or a subclass' often gives false positives at dev time
+@SuppressWarnings("squid:MaximumInheritanceDepth", "squid:S2166")
+public class $EXCEPTION_CLASS_NAME extends CodedRuntimeException
```

Comment:
CodedRuntime is currently not handledby the GlobalExceptionHandler, maybe replace with a placeholder and a comment that states the exceptions that are handled?